Resources/JavaScript/script.js

```diff
@@ -0,0 +1,90 @@
+"use strict";
```

Comment:
What does this do?

Reply:
I use "use strict" to monitor and flag spelling errors. Like if I declare a variable like, 'let spellingError' and by mistakenly use 'spelingError', it will flag. I learnt it from Jonas Schmedtmann before joining NextGen.